[OCPBUGS#34889]: Add warning about rebooting nodes (#110736)
openshift/openshift-docs: updating/updating_a_cluster/updating-hardware-on-nodes-running-on-vsphere.adoc

* Before upgrading OpenShift 4.12 to OpenShift 4.13, you must update vSphere to *v8.0 Update 1 or later*; otherwise, the OpenShift 4.12 cluster is marked *un-upgradeable*.
====

[WARNING]
====

> Updating custom API certificates triggers a rolling reboot of the control plane nodes via the Machine Config Operator. These nodes must be updated serially. Ensure each node returns to a Ready state and the `etcd` static pods are healthy before the next node in the sequence begins its update. Failure to do so may result in a loss of etcd quorum and cluster-wide downtime.

Reviewer: Try to avoid stacked notes. Maybe move the "You can update your virtual hardware immediately or schedule an update in vCenter." sentence above your note?

<img width="1001" height="164" alt="Image" src="https://github.com/user-attachments/assets/f233905c-e6b0-4abb-92f1-2e96c01a4685" />

> Updating custom API certificates triggers a rolling reboot of the control plane nodes via the Machine Config Operator. These nodes must be updated serially. Ensure each node returns to a Ready state and the `etcd` static pods are healthy before the next node in the sequence begins its update. Failure to do so may result in a loss of etcd quorum and cluster-wide downtime.

Reviewer: ```suggestion
Updating custom API certificates triggers the Machine Config Operator (MCO) to initiate a rolling reboot of the control plane nodes. These nodes must be updated serially. Ensure each node returns to a `Ready` state and the `etcd` static pods are healthy before the next node in the sequence begins its update. Failure to do so might result in a loss of etcd quorum and cluster-wide downtime.
```

<img width="1442" height="117" alt="Image" src="https://github.com/user-attachments/assets/72f32784-e9f5-4122-90dd-57faf509168e" />

<img width="1442" height="117" alt="Image" src="https://github.com/user-attachments/assets/1780e222-ad5c-40ef-860b-6bf1622adfa2" />

The adding-a-custom-CA-to-a-running-cluster.adoc file mentions "initiating a rolling reboot"
